doc/pylint.rst

```diff
@@ -1,36 +1,97 @@
 All standard checks are run by default.
-You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective level, where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use error codes like ``C0321`` although names are preferred for clarity).
+To go directly to line in the :doc:`editor` highlighted by a failed check, just click its name.
+
+.. image:: images/pylint/code_analysis_editor.gif
+   :alt: Spyder Pylint pane, showing running analysis and clicking failed check
+
+|
+
+You can manually enter the Python module or package path you'd like it to check in the path entry textbox in the :guilabel:`Code Analysis` pane. 
+The analyzer works with both ``.py`` (or ``.pyw``) Python scripts and whole Python packages (directories containing an :file:`__init__.py` file).
+
+.. image:: images/pylint/code_analysis_file.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why. 
+
+.. image:: images/pylint/code_analysis_stop.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses; 
+
+
+
+===================
+Contex menu options
+===================
+The number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog from the :guilabel:`Code Analysis` context menu.
+
+.. image:: images/pylint/code_analysis_history.gif
+   :alt: Spyder Pylint pane, customizing history dialog
+
+|
+
+You can also expand or collapse one or all the sections in the pane by using the corresopnding options in the context menu.
+
+.. image:: images/pylint/code_analysis_expand_collapse.gif
+   :alt: Spyder Pylint pane, customizing history dialog
+
+|
+
+
+
+================
+Advanced options
+================
+
+You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective `level`_, (LINK) where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use `error codes`_ like ``C0321`` although names are preferred for clarity).  
+
+.. _error codes: http://pylint.pycqa.org/en/latest/technical_reference/features.html#pylint-checkers-options-and-switches
+
+.. _level: http://pylint.pycqa.org/en/latest/user_guide/message-control.html
+
+.. image:: images/pylint/code_analysis_pylint_disable.gif
```

Comment:
Can be sped up by ~1.5x, but otherwise looks great.

Reply:
I don't agree with using a gif here because it's very difficult to understand what's happening.

Instead, I'd suggest to replace it with a still png that shows a simple error in a very simple code in the editor on the left and how that error is ignored by Pylint in the Code Analysis pane on the right.